clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/deploy/DescriptionAuthorizer.java

```diff
@@ -74,19 +76,25 @@ public void authorize(T description, Errors errors) {
     String account = null;
     List<String> applications = new ArrayList<>();
     boolean requiresApplicationRestriction = true;
-    boolean requiresAuthentication = true;
 
     if (description instanceof AccountNameable) {
       AccountNameable accountNameable = (AccountNameable) description;
-      account = accountNameable.getAccount();
 
       requiresApplicationRestriction = accountNameable.requiresApplicationRestriction();
-      requiresAuthentication = accountNameable.requiresAuthentication(opsSecurityConfigProps);
```

Comment:
This boolean had to go -- I don't want someone to use it later in the code and then accidentally skip a resource that requires authorization.